packages/jest-cli/src/lib/__tests__/PromptController-test.js

```diff
@@ -0,0 +1,66 @@
+/**
+* Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree. An additional grant
+* of patent rights can be found in the PATENTS file in the same directory.
+*
+* @emails oncall+jsinfra
+*/
+'use strict';
+
+const PromptController = require('../PromptController');
+const {KEYS} = require('../../constants');
+
+it('should calls handler on change value', () => {
+  const promptController = new PromptController();
+  const onChange = jest.fn();
+
+  promptController.prompt(onChange, jest.fn(), jest.fn());
+
+  expect(onChange).toHaveBeenLastCalledWith('');
+
+  promptController.put('74');
+  expect(onChange).toHaveBeenLastCalledWith('t');
+
+  promptController.put('65');
+  expect(onChange).toHaveBeenLastCalledWith('te');
+
+  promptController.put('73');
+  expect(onChange).toHaveBeenLastCalledWith('tes');
+
+  promptController.put('74');
+  expect(onChange).toHaveBeenLastCalledWith('test');
+
+  expect(onChange).toHaveBeenCalledTimes(5);
+});
+
+it('should calls handler on success prompt', () => {
+  const promptController = new PromptController();
+  const onSuccess = jest.fn();
+
+  promptController.prompt(jest.fn(), onSuccess, jest.fn());
+
+  promptController.put('74');
```

Comment:
I think we could extract this to a `KEYS` object at the top level of this file, this should make the tests easier to follow in the future :) 

```javascript
let {KEYS} = require('../../constants');
KEYS = Object.assign({}, KEYS, {
  E: '65',
  S: '73',
  T: '74',
}
```

```javascript
promptController.put(KEYS.T);
```